themes/luci-theme-bootstrap/htdocs/luci-static/bootstrap/cascade.css

```diff
@@ -1448,7 +1452,7 @@ body.modal-overlay-active #modal_overlay {
 }
 
 .btn .close, .alert-message .close {
-	font-family: Arial, sans-serif;
+	font-family: var(--font-sans);
```

Comment:
You are changing the current font:
`
--font-sans: ui-sans-serif, system-ui, sans-serif;
`